cppcache/integration/test/CacheXmlTest.cpp

```diff
@@ -59,9 +57,23 @@ apache::geode::client::Cache createCacheUsingXmlConfig() {
  * furtures.
  */
 TEST(CacheXmlTest, loadCacheXml) {
-  Cluster cluster{LocatorCount{1}, ServerCount{1}};
+  Cluster cluster{LocatorCount{0}, ServerCount{0}};
```

Comment:
Pretty sure you can just leave the `LocatorCount` and `ServerCount` off. It should default to 0.

Reply:
> Pretty sure you can just leave the `LocatorCount` and `ServerCount` off. It should default to 0.

Turns out no, this attempts to invoke the copy ctor and breaks, BUT... it's possible to side-step the issue by just removing the cluster from the tests altogether, since it turns out to not be needed.